rules/sinks/third_parties/sdk/amazon/java.yaml

```diff
@@ -9,13 +9,13 @@ sinks:
     domains:
       - "amazon.com"
     patterns:
-      - "(?i)(com[.]amazon[.]alexa|com[.]amazon[.]android|com[.]amazon[.]redshift|software[.]amazon[.]awscdk|software[.]amazon[.]awsconstructs|software[.]amazon[.]awssdk|software[.]amazon[.]ion|software[.]amazon[.]jsii|software[.]amazon[.]kinesis|software[.]amazon[.]konstruk|software[.]amazon[.]smithy).*"
```

Comment:
`software[.]amazon[.]awscdk|software[.]amazon[.]awsconstructs` should these also be removed ??

Reply:
Updated the rules.